src/app/components/mno-app-install-btn/mno-app-install-btn.coffee

```diff
@@ -202,6 +200,8 @@ angular.module 'mnoEnterpriseAngular'
             # Is the product externally provisioned
             vm.isExternallyProvisioned = vm.isProvisioningEnabled && (product?.product_type == 'application' || product?.externally_provisioned)
 
+            vm.orderPossible = plans[currency] || !ProvisioningHelper.hasPricedPlan(plans.default) || ProvisioningHelper.skipPriceSelection(product)
+
```

Comment:
I do see an issue with this logic. Lets say I have an app that has default plans that include a priced plan as well as a usage based plan. The call to 
 `ProvisioningHelper.hasPricedPlan(plans.default)` will return true even though there is a plan that is not priced. I actually think that `vm.orderPossible` should be true for the following conditions:
```
vm.orderPossible = plans[currency] || plans.default[0] || ProvisioningHelper.skipPriceSelection(product)
```
This way as long as there is a default plan the button will be enabled. It will say no plans available in your currency but once you click provision you can select from what is available.